src/components/Items/Question/Footer.vue

```diff
@@ -0,0 +1,192 @@
+<template>
+  <div class="flex w-full bg-white p-1 md:p-3 justify-between">
+    <div class="place-self-start flex h-full relative">
+      <!-- revise button -->
+      <!-- using invisibility to hide buttons here instead of using class hidden
+           to ensure that the fullscreen button is at the center in mobile view -->
+      <icon-button
+        :titleConfig="reviseButtonTitleConfig"
+        :buttonClass="reviseButtonClass"
+        class="btn"
+        :class="{ invisible: isAnswerSubmitted }"
+        @click="reviseClicked"
+      ></icon-button>
+      <!-- icon to show correct/wrong option result -->
+      <inline-svg
+        :src="answerCorrectnessIcon"
+        :class="answerCorrectnessIconClass"
+        class="w-6 h-6 sm:w-10 sm:h-10 lg:w-12 lg:h-12 place-self-center ml-4 absolute"
+      ></inline-svg>
+    </div>
+    <!-- button to enter/exit fullscreen -->
+    <icon-button
+      :iconConfig="fullscreenIconConfig"
+      :buttonClass="fullscreenButtonClass"
+      @click="toggleFullscreen"
+    ></icon-button>
```

Comment:
Would recommend adding some text here "go fullscreen" 
or/and change the fullscreen icon
-- it was not obvious for me until it was shown to me

Reply:
haan - we'll have to do usability testing - abhi not sure what is the best way - Will need to look at responsiveness etc as well for text there - let me see how easy/hard it is.

Reply:
Or use YT's fullscreen icons